pkg/storage/types/pvc.go

```diff
@@ -46,6 +54,14 @@ func (e PvcNotFoundError) Error() string {
 	return e.Reason
 }
 
+func IsPVCOwnedByVirtualMachine(pvc *k8sv1.PersistentVolumeClaim) bool {
```

Comment:
We should check if the pvc is nil before accessing the annotations

Reply:
Should never be nil:  https://github.com/mhenriks/kubevirt/blob/dvt-claim-adoption/pkg/virt-controller/watch/vm.go#L467-L477

Is there a standard for this I don't like adding lots of nil checks?

Reply:
Well, you never know who is going to call this function, especially if it is going to be reused int he future